$OpenBSD: patch-plugins_project-wizard_templates_pygtk_wiz_in,v 1.2 2011/10/04 11:24:41 jasper Exp $

Prevent an error saying that python isn't installed by adjusting
the python binary name.

--- plugins/project-wizard/templates/pygtk.wiz.in.orig	Tue Oct  4 13:09:50 2011
+++ plugins/project-wizard/templates/pygtk.wiz.in	Tue Oct  4 13:10:05 2011
@@ -8,7 +8,7 @@
 	    <required-program>automake</required-program>
 	    <required-program>autoconf</required-program>
 	    <required-program>make</required-program>
-	    <required-program>python</required-program>
+	    <required-program>${MODPY_BIN}</required-program>
 	    <required-package>gtk+-3.0</required-package>
 	    <!-- Not working: <required-package>python-gtk2</required-package>-->
     </project-wizard>
